utils/data-processing-family-task-submitter/pom.xml

```diff
@@ -94,14 +94,14 @@
             <artifactId>codec-json</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.hpe.caf</groupId>
-            <artifactId>caf-api</artifactId>
-            <version>12.0.0-SNAPSHOT</version>
+            <groupId>com.github.workerframework</groupId>
+            <artifactId>worker-api</artifactId>
+            <version>2.0.0-279</version>
```

Comment:
I think this should still be caf-api;
https://github.com/CAFapi/caf-common/blob/develop/caf-api/pom.xml